src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -28,7 +28,6 @@
           <div class="kuiLocalSearch">
             <input
               parse-query
-              input-focus
```

Comment:
Hmm, I'm not a huge fan of this. Now, when Discover loads, I can't get to work immediately without using my mouse to focus in on the search input.

What if, instead, we put a help icon to the right side of the search, and when you clicked on it, it brought up this same help dialog?

/cc @cjcenizal